index.bs

```diff
@@ -91,6 +91,10 @@ spec: credential-management-1; urlPrefix: https://w3c.github.io/webappsec-creden
     for: CredentialsContainer
         type: method
             text: create(); url: dom-credentialscontainer-create
+
+spec: mixed-content; urlPrefix: https://w3c.github.io/webappsec-mixed-content/
```

Comment:
mixed-content is recently at CR, should we refer to that?
https://www.w3.org/TR/mixed-content/

Reply:
I am not too familiar with the TPAC decisions. My read was that the three approaches are all options but there was no decision. Please correct me if I am wrong. The current spec doesn't have any restriction in this area. Adding the restriction of  "priori authenticated URL" is more or less an ad hoc approach anyway because otherwise the page should show mixed content warning. 

Can you please explain why there should be a corresponding CTAP change? I don't see a mention about converting URL before sending to CTAP in the spec today. 

I updated the PR to reference the CR draft.